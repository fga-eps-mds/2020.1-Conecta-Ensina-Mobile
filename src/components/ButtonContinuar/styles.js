import styled from 'styled-components';

export const Container = styled.TouchableOpacity`
  background-color: ${(props) => props.theme.colors.azulClaro};
  width: 138px;
  height: 31px;
  border-radius: 8px;
  align-items: center;
  justify-content: center;
  margin-top: 440px;
  align-self: center;
`;
