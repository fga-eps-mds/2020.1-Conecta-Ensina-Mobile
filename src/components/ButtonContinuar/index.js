import React from 'react';
import {Button} from './styles';
import CustomText from '../CustomText';
import Theme from '../../../Theme';

export default function ButtonContinuar({onPress, children}) {
  return (
    <Theme>
      <Button onPress={onPress}>
        <CustomText white bigSmall>
          {children}
        </CustomText>
      </Button>
    </Theme>
  );
}
