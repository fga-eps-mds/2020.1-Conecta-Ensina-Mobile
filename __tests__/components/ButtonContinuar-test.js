import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';

import ButtonContinuar from '../../src/components/ButtonContinuar';

it('renders correctly', () => {
  const tree = renderer.create(<ButtonContinuar />).toJSON();
  expect(tree).toMatchSnapshot();
});
