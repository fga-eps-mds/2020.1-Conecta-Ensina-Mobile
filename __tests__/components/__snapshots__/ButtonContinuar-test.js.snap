// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#297FB8",
      "borderRadius": 8,
      "height": 31,
      "justifyContent": "center",
      "marginTop": 440,
      "opacity": 1,
      "width": 138,
    }
  }
>
  <Text
    bigSmall={true}
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontFamily": "Roboto",
          "fontSize": 16,
        },
      ]
    }
    white={true}
  />
</View>
`;
